### ISSUE TITLE
Multiple Maps At Once

### ISSUE THREAD

**Issue by Crash370** [Durtur/Dungeoneer#22]
It would be awesome if we could open multiple maps at once - I play DnD on zoom and being able to flick between maps as needed would be super helpful.

**Comment by Durtur** [comment]
@Crash370  there's an issue for multi level map support: https://github.com/Durtur/Dungeoneer/issues/13. Do you want to be able to have multiple levels of the same map or is this some sort of quick select you are thinking about?

**Comment by Crash370** [comment]
More like being able to open multiple map windows at once - so if I'm streaming online say, I could flick between them when needed.

**Comment by StumbleRunner** [comment]
Yes, I had a similar issue where the players were, for a time moving back and forth, between two dungeon levels...
